gtest_parallel.py

```diff
@@ -703,16 +714,17 @@ def main():
   assert len(unique_binaries) == len(binaries), (
       "All test binaries must have an unique basename.")
 
-  # Remove files from old test runs.
-  if os.path.isdir(options.output_dir):
-    shutil.rmtree(options.output_dir)
-  # Create directory for test log output.
-  try:
-    os.makedirs(options.output_dir)
-  except OSError as e:
-    # Ignore errors if this directory already exists.
-    if e.errno != errno.EEXIST or not os.path.isdir(options.output_dir):
-      raise e
+  if options.output_dir is not None:
```

Comment:
This can be `if options.output_dir:` right?

Reply:
I think '' shouldn't be accepted as a command-line parameter.

Reply:
Should we check that `output_dir` exists and reset it to `None` if it doesn't?

Reply:
I think we should reject the parameter it if the directory doesn't exist, can we validate this after parser.parse_args()?